pallets/runtime/tests/src/pips_test.rs

```diff
@@ -1235,9 +1236,6 @@ fn assert_pruned(id: PipId) {
     assert_eq!(Pips::proposals(id), None);
     assert_vote_details(id, VotingResult::default(), vec![], vec![]);
     assert_eq!(Pips::pip_to_schedule(id), None);
-    for v in <pallet_pips::ExecutionSchedule<TestStorage>>::iter_values() {
-        assert!(v.iter().all(|x| *x != id));
-    }
```

Comment:
Can you replace this with an equivalent check?

Reply:
We can only query the number of scheduled calls, not the scheduled calls themselves. So there is no equivalent check.

Reply:
I created an [issue](https://github.com/paritytech/substrate/issues/7449) for that.

Reply:
Ah; thanks! Can you leave a TODO for the time being + reference the substrate issue there?